### ISSUE TITLE
Run colima VM on a server

### ISSUE THREAD

**Issue by iosifnicolae2** [abiosoft/colima#1077]
### Description

It would be really useful if we could run the Colima VM on an existing server, whether in a VM in the cloud or on-premise.

**Comment by abiosoft** [comment]
It is definitely possible. Can you be more specific with your scenario?

**Comment by iosifnicolae2** [comment]
I'm using an M1 with just 16GB of RAM, and I would like to speed things up by building and running all Docker images on a more powerful remote server with more memory.

**Comment by abiosoft** [comment]
You mean, you want Colima to manage docker running on remote machines?

**Comment by iosifnicolae2** [comment]
Yes, instead of launching the VM on my machine, I would like to be able to pass an SSH endpoint like ssh://ubuntu@XXXX, and Colima will set up everything there.

**Comment by RouquinBlanc** [comment]
That would indeed be super cool, although then there would be nothing left of 'lima' in that use of Colima 😅 

I have the same need: Using something like colima to mount automatically the docker socket, k3s context and local directory forwarding, but with a remote powerful server, leaving my laptop not burning and swapping 😄

**Comment by abiosoft** [comment]
I have thought of this idea, but I would probably make it a separate project.
The challenge with a remote machine is always volume mounts, people expect it to behave like a local directory with regards to the performance.
